migrate-the-clay-components-from-v2-to-v3.md

```diff
@@ -139,6 +139,45 @@ To get to the behavior of having a ClayLink with image, use the composition with
 +</ClayLink>
 ```
 
+## ClayNavigationBar
+
+ClayNavigationBar has become simpler than the version `v2`, removing APIs from `items` and turning more flexible to pass elements by composition. Turning the development more easier when defining your Dropdown's list items but complying with Lexicon specifications.
```

Comment:
I do not know if we would say that we are more in compliance with the Lexicon, since we are giving more flexibility, we are leaving slightly off the curve with the specifications.